web/src/components/PoiDetails.tsx

```diff
@@ -225,10 +226,19 @@ const PoiDetails: React.FC<PoiDetailsProps> = ({
           </Collapsible>
         </>
       )}
+      <>
```

Comment:
I think this should depend on whether we actually have opening hours.

Reply:
I got the point but i would have to pull all the logic and template for the title out of the component into `PoiDetails` since even we don't get `openingHours` we may render the `titleContainer`